SnapshotTests/Changes.diff

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
Is this file needed?

Reply:
fixed [988c3dc](https://github.com/salemove/ios-sdk-widgets/pull/521/commits/988c3dc34073022cd6df7569db79d476a7a6646f)